website/versions.json

```diff
@@ -1,4 +1,5 @@
 [
+  "v2.0.8",
```

Comment:
In this instance I triple-verified the version up script did not create a previous doc.